src/user-guide/writing-suites/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _User Guide Configuration:
+
+Workflow Configuration
+======================
+
+Cylc suites are defined in structured, validated, :cylc:conf:`flow.cylc`
+files that concisely specify the properties of, and the relationships
+between, the various tasks managed by the suite.
+
+Here we will look at the format and content of the
+:cylc:conf:`flow.cylc` file and how to configure suites.
```

Comment:
```suggestion
Here we will look at:
- Folders which may (optionally) accompany a workflow configuration.
- The contents of the :cylc:conf:`flow.cylc` file.
- How to configure suites.
```

Reply:
This is just a file that I split into multiple parts, I didn't actually edit this but since you've gone to the effort to read it (and hats off for that) might as well take a look in.